admin_guide/high_availability.adoc

```diff
@@ -252,23 +282,95 @@ ip addr show $interface | grep -i MULTICAST
 echo "Check multicast groups ... "
 ip maddr show $interface | grep 224.0.0 | grep $interface
 
-echo "Optionally, add accept rule and persist it ... "
-sudo /sbin/iptables -I INPUT -i $interface -d 224.0.0.18/32 -j ACCEPT
-
-echo "Please ensure the above rule is added on system restarts."
-
 EOF
 done;
 ----
 ====
 
-. Depending on your environment policies, you can either reuse the *router*
-service account created previously or create a new *ipfailover* service account.
+[[options-environment-variables]]
+=== Command Line Options and Environment Variables
+
+.Command Line Options and Environment Variables
+[cols="1a,3a,1a,4a",options="header"]
+|===
+
+| Option | Variable Name | Default | Notes
```

Comment:
It is the default value of the environment variable that is in the DC. The pod start up with the environment variables and processes based on them. What are you thinking about here?